Formula/libpython-tabulate.rb

```diff
@@ -0,0 +1,53 @@
+class LibpythonTabulate < Formula
+  desc "Pretty-print tabular data in Python"
+  homepage "https://pypi.org/project/tabulate/"
+  url "https://files.pythonhosted.org/packages/ae/3d/9d7576d94007eaf3bb685acbaaec66ff4cdeb0b18f1bf1f17edbeebffb0a/tabulate-0.8.9.tar.gz"
+  sha256 "eb1d13f25760052e8931f2ef80aaf6045a6cceb47514db8beab24cded16f13a7"
+  license "MIT"
+
+  keg_only <<~EOS
+    this formula is mainly used internally by other formulae.
+    Users are advised to use `pip` to install python-tabulate
+  EOS
```

Comment:
This won't work cleanly. It is the linking that allows dependents to find Python package due to default site-packages.

If you want to keep keg-only, you will need to create .pth files for every dependent's venv so it can find `tabulate`.

The easier option is to follow `six` as an example and link formula.

Reply:
Done